client/apps/edit/components/header2/index.jsx

```diff
@@ -0,0 +1,188 @@
+      return 'Saving...'
+    } else if (article.get('published')) {
+      return 'Save Article'
+    } else {
+      return 'Save Draft'
+    }
+  }
+
+  getPublishText = () => {
+    const { article, edit } = this.props
+    const { isPublishing } = edit
+    const isPublished = article.get('published')
+
+    if (isPublishing && isPublished) {
+      return 'Publishing...'
+    } else if (isPublishing) {
+      return 'Unpublishing...'
+    } else if (isPublished) {
+      return 'Unpublish'
+    } else {
+      return 'Publish'
+    }
+  }
+
+  contentIsComplete = () => {
+    const { article } = this.props
+    return article.get('title') && article.get('title').length
+  }
+
+  displayIsComplete = () => {
+    const { article } = this.props
+    return article.get('thumbnail_title') && article.get('thumbnail_image')
+  }
+
+  render () {
+    const { article, actions, channel, user } = this.props
+    const { activeSection, isDeleting } = this.props.edit
+
+    return (
+      <div className='EditHeader'>
+
+        <div className='EditHeader__left'>
+          <div className='EditHeader__tabs'>
+            <button
+              className='avant-garde-button check'
+              onClick={() => this.toggleSection('content')}
+              data-active={activeSection === 'content'}
+            >
+              <span>Content</span>
+              <Icon
+                className='icon'
+                name='check'
+                color={this.contentIsComplete() ? 'limegreen' : '#ccc'}
+              />
+            </button>
+
+            <button
+              className='avant-garde-button check'
+              onClick={() => this.toggleSection('display')}
+              data-active={activeSection === 'display'}
+            >
+              <span>Display</span>
+              <Icon
+                className='icon'
+                name='check'
+                color={this.displayIsComplete() ? 'limegreen' : '#ccc'}
+              />
+            </button>
+
+            {user.type === 'Admin' &&
+              <button
+                className='avant-garde-button'
+                onClick={() => this.toggleSection('admin')}
+                data-active={activeSection === 'admin'}
+              >
+                Admin
+              </button>
+            }
+          </div>
+
+          <div>
+            <button
+              className='avant-garde-button publish'
+              data-disabled={!this.isPublishable()}
+              onClick={this.onPublish}
+            >
+              {this.getPublishText()}
+            </button>
+
+            {channel.get('type') === 'editorial' &&
+              <button
+                className='avant-garde-button autolink'
+              >
+                Auto-link
+              </button>
+            }
+          </div>
+        </div>
+
+        <div className='EditHeader__right'>
+          <button
+            className='avant-garde-button'
+            style={{border: 'none'}}
```

Comment:
The styling seems to be pulling in different directions. In this file alone, I'm seeing three different ways of styling. There's our legacy .styl, then we have this `style={}` format, and then finally we have an Icon that uses styled-components. I feel like at this point for new code we should be sticking to one or two ways.